task-sdk/src/airflow/sdk/execution_time/task_runner.py

```diff
@@ -677,6 +684,35 @@ def startup() -> tuple[RuntimeTaskInstance, Context, Logger]:
         ti.log_url = get_log_url_from_ti(ti)
     log.debug("DAG file parsed", file=msg.dag_rel_path)
 
+    try:
+        run_as_user = getattr(ti.task, "run_as_user", None) or conf.get("core", "default_impersonation")
+    except AirflowConfigException:
+        run_as_user = None
```

Comment:
Maybe instead of try/except here (mostly to avoid one extra import across sdk/core)

```suggestion
    run_as_user = getattr(ti.task, "run_as_user", None) or conf.get("core", "default_impersonation", fallback=None)
```

Reply:
Yeah not a bad idea

Reply:
Done.